package.json

```diff
@@ -20,7 +20,7 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^6.2.x",
     "babel-preset-es2015": "^6.24.1",
-    "backbone": "~1.1.2",
+    "backbone": "~1.3.3",
```

Comment:
@danlamanna I totally agree. I'll be trying to incorporate new opportunities in other PRs, but I'll be happy for now if there are no breakages.